src/docmanager/context.ts

```diff
@@ -528,6 +535,20 @@ class ContextManager implements IDisposable {
   }
 
   /**
+   * Resolve a relative url to a correct server path.
+   */
+  resolveUrl(id: string, url: string): string {
+    // TODO: use proper url parser here.
+    if (url.indexOf('http') === 0 || url.indexOf('data') === 0) {
```

Comment:
Let's just check for a colon for now, there are other API changes coming from an upgrade to js-services that should be done in a follow-on PR.